File: Day_3/EmployeeManagement/src/main/java/com/demo/Employee.java
package com.demo;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
//@AllArgsConstructor
public class Employee {
	
	private int id;
	private String name;
	private int age;
	
	public Employee() {
		System.out.println("Employee Default Constructor");
	}
	
	

}

File: Day_8/ReactApplication/reactapp/src/component/Employee.js
import React, { Component } from 'react'
import Skill from './Skill'

export class Employee extends Component {
    constructor(props) {
        super(props)
    }
    render() {
        return (
            <div>
                <b>{this.props.id}</b><br />
                <b>{this.props.name}</b><br />
                <b>{this.props.age}</b><br />
                <b><Skill technology="Java" /></b>
            </div>
        )
    }
}

export default Employee